### PR TITLE
Don't cast the autocrypt setup message verification key parts into numbers

### PR DESCRIPTION
**Pull request by jankass**

### CODE REVIEW

File: src/renderer/components/dialogs/SetupMessage.js [hunk 1]
```diff
@@ -19,16 +19,18 @@ class SetupMessagePanel extends React.Component {
...
+    if (value.length > 4 || isNaN(valueNumber) || valueNumber < 0 || valueNumber > 9999) return false
+    
     let updatedkey = this.state.key
     updatedkey[Number(event.target.id)] = value
     this.setState({ key: updatedkey })
   }
 
   onClick (event) {
     const autocryptKey = this.state.key.join('')
+    console.log(autocryptKey)
```
Comment (ralphtheninja) [inline src/renderer/components/dialogs/SetupMessage.js]: Remove this please.

**Comment by ralphtheninja** [comment]
Oh, there's some `standard` left.

**Comment by ralphtheninja** [comment]
@jankass This fixes #280 right?

**Comment by karissa** [comment]
nice

**Comment by r10s** [comment]
+1 :)

**Pull request merged**